spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -802,7 +802,7 @@ skip_inc_sync("CP/DP labels #" .. strategy, function()
   end)
 end)
 
-skip_inc_sync("CP/DP cert details(cluster_mtls = shared) #" .. strategy, function()
+pending_inc_sync("CP/DP cert details(cluster_mtls = shared) #" .. strategy, function()
```

Comment:
We must fix it when enabling incremental sync

Reply:
This PR aims to keep an accurate status of the tests rather than making them a permanent state